[2025-07-26T09:10:50.429+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-26T09:10:50.515+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_postgres2.load_user_dim_table scheduled__2025-07-25T10:00:00+00:00 [queued]>
[2025-07-26T09:10:50.548+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_postgres2.load_user_dim_table scheduled__2025-07-25T10:00:00+00:00 [queued]>
[2025-07-26T09:10:50.551+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-07-26T09:10:50.616+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_user_dim_table> on 2025-07-25 10:00:00+00:00
[2025-07-26T09:10:50.660+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=291) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T09:10:50.669+0000] {standard_task_runner.py:72} INFO - Started process 299 to run task
[2025-07-26T09:10:50.664+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_postgres2', 'load_user_dim_table', 'scheduled__2025-07-25T10:00:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpsq7dbwfn']
[2025-07-26T09:10:50.681+0000] {standard_task_runner.py:105} INFO - Job 151: Subtask load_user_dim_table
[2025-07-26T09:10:50.995+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_postgres2.load_user_dim_table scheduled__2025-07-25T10:00:00+00:00 [running]> on host 72bd964f8e11
[2025-07-26T09:10:51.919+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_postgres2' AIRFLOW_CTX_TASK_ID='load_user_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2025-07-25T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-25T10:00:00+00:00'
[2025-07-26T09:10:51.926+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-26T09:10:51.986+0000] {etl_dag.py:312} INFO - Loading user dimension table
[2025-07-26T09:10:51.997+0000] {etl_dag.py:332} INFO - Inserted 0 records into users table
[2025-07-26T09:10:52.003+0000] {etl_dag.py:335} INFO - User dimension table loaded successfully
[2025-07-26T09:10:52.007+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-26T09:10:52.069+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-26T09:10:52.071+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_postgres2, task_id=load_user_dim_table, run_id=scheduled__2025-07-25T10:00:00+00:00, execution_date=20250725T100000, start_date=20250726T091050, end_date=20250726T091052
[2025-07-26T09:10:52.162+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-26T09:10:52.235+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-26T09:10:52.240+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
